tests/search/test_SwapStar.py

```diff
@@ -37,15 +38,13 @@ def test_swap_star_identifies_additional_moves_over_regular_swap(rc208):
     ls = LocalSearch(rc208, rng, compute_neighbours(rc208, nb_params))
 
     ls.add_node_operator(Exchange11(rc208))
-    ls.add_route_operator(SwapStar(rc208))
+    ls.add_route_operator(SwapStar(rc208, 1))
```

Comment:
```suggestion
    ls.add_route_operator(SwapStar(rc208, overlap_tolerance=1))
```
(here, and other places in this file)